BuildResidentialScheduleFile/resources/schedules.rb

```diff
@@ -867,7 +867,7 @@ def export(schedules_path:)
   # @param min [TODO] TODO
   # @param max [TODO] TODO
   # @return [TODO] TODO
-  def gaussian_rand(prng, mean, std, min = nil, max = nil)
+  def gaussian_rand(prng, mean, std, min = nil, max = 0.1)
```

Comment:
You were supposed to update the min. :)

Reply:
Oops, my fault, I said max instead of min. Sorry Joe.

Reply:
Ha! Oops.